templates/confluence.service.erb

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Confluence Team Collaboration Software
+After=network.target
+
+[Service]
+Type=forking
+User=confluence
+PIDFile=<%= scope.lookupvar('confluence::webappdir') %>/work/catalina.pid
```

Comment:
Seeing as the head of the tree is broken on EL7 because this unit file is missing, I would say yes, it is needed. Even if the head of the tree was reverted to use the init script on EL7, there is still a problem as described in #19 where systemctl falls back to using chkconfig/service commands and attempts to enable the service on every puppet run.